doc/snippets/Microsoft.Data.SqlClient/SqlAuthenticationProvider.xml

```diff
@@ -1,22 +1,73 @@
-<docs>
+﻿<docs>
   <members name="SqlAuthenticationProvider">
     <SqlAuthenticationProvider>
-      <summary>Defines the core behavior of authentication providers and provides a base class for derived classes. </summary>
+      <summary>Defines the core behavior of authentication providers and provides a base class for derived classes.</summary>
       <remarks>
-        <format type="text/markdown">
-          <![CDATA[  
-  
-## Remarks  
- Derived classes must provide a parameterless constructor if they can be instantiated from the app.config file.
-
-## Examples  
- The following example demonstrates implementing a custom SqlAuthenticationProvider and providing the same to SqlClient for overriding Device Code Flow authentication mode:
- 
- [!code-csharp[ActiveDirectory_DeviceCodeFlowCallback Example#1](~/../sqlclient/doc/samples/CustomDeviceCodeFlowAzureAuthenticationProvider.cs#1)]
-  
- ]]>
-        </format>
+        Derived classes must provide a parameterless constructor if they can be instantiated from the app.config file.
       </remarks>
+      <example>
+        The following example demonstrates implementing a custom SqlAuthenticationProvider and providing the same to SqlClient for overriding Device Code Flow authentication mode:
+        <code language="c#">
+          <![CDATA[
```

Comment:
we do not need the CDATA here anymore. if you want to separate a part of the text as a paragraph use <para> tag instead, but in this specific case we wont need it.

Reply:
This was based on what I saw the Cosmos team using (eg, https://github.com/Azure/azure-cosmos-dotnet-v3/blob/master/Microsoft.Azure.Cosmos/src/CosmosClient.cs) 

My thought was that the CDATA is required to preserve the whitespace/indentation since xml ignores >1 space. I'll double check though. I don't particularly like using CDATA blocks, so if I can avoid them, I'll be happier.

Reply:
Double checked, yep, the get processed just fine without the CDATA block. I've gone ahead and removed them and applied the &lt; &gt; changes required to make it valid.